.eslintrc

```diff
@@ -94,3 +94,4 @@ rules:
   "no-path-concat": 0
   "no-process-exit": 0
   "eol-last": 2
+  "key-spacing": 0
```

Comment:
In some cases, I actually find that adding a few spaces after the semi column can make sense. [I've updated the PR](https://github.com/mozilla-services/loop-server/commit/9f260b1aa397f129bbf5bb6f5670fcc1a002482e) to address the key-spacing warnings, but I cannot really tell if I prefer the old or the new version.

Since eslint thinks its best, I think I'll align with them there, conventions are good.